zaas-service/src/main/java/org/zowe/apiml/zaas/config/SwaggerConfig.java

```diff
@@ -23,7 +24,8 @@
 import java.net.URI;
 import java.util.ArrayList;
 
-@Configuration
+@Configuration("zaasSwaggerConfig")
+@ConditionalOnMissingBean(name = "modulithConfig")
```

Comment:
Are the endpoints in the new OpenAPI docs the same as in the original one? See the definition in the file, for example.

Reply:
done